pyglmnet/utils.py

```diff
@@ -130,3 +137,26 @@ def set_log_level(verbose):
             raise ValueError('verbose must be of a valid type')
         verbose = logging_types[verbose]
     logger.setLevel(verbose)
+
+
+def verbose_iterable(data):
```

Comment:
private function? we don't want to expose this in the public API, do we?

Reply:
mmh how can I make this thingy private? I remember you can only do that for class methods. Anyhow, I changed the method's signature to `_verbose_iterable` to make it less public-looking.